db/seeds/run-seed.js

```diff
@@ -1,4 +1,4 @@
-const devData = require('../data/development-data/index.js');
+const devData = require('../data/test-data/index.js');
```

Comment:
This should stay as devData, otherwise you'll be changing this file back and fourth between test and dev data